### PR TITLE
docs: dotnet Uno in ArchLinux

### PR DESCRIPTION
**Pull request by Arwwarrr**
and other distributions that are based on archlinux: Manjaro

### CODE REVIEW

File: doc/articles/get-started-with-linux.md [hunk 1]
```diff
@@ -77,3 +77,46 @@ Now let's run the application:
     cd MyUnoApp.Skia.Gtk
     dotnet run
     ```
+    
+    
+    
+Using ArchLinux 5.8.14 or later / Manjaro:
```
Comment (jeromelaban) [inline doc/articles/get-started-with-linux.md:83]:
```suggestion
## Setting up for ArchLinux 5.8.14 or later / Manjaro:
```

File: doc/articles/get-started-with-linux.md [hunk 2]
```diff
@@ -77,3 +77,46 @@ Now let's run the application:
...
+    ```
+    
+turns on the browser and type
+```
+http://localhost:5000/
+```  
+    
+    
+    
+
```
Comment (jeromelaban) [inline doc/articles/get-started-with-linux.md:122]:
```suggestion
```

File: doc/articles/get-started-with-linux.md [hunk 3]
```diff
@@ -77,3 +77,46 @@ Now let's run the application:
...
+    ```
+
+Now let's run the application:
+- Open the folder created by `dotnet new`
+- In the terminal, build and run the application:
+    ```bash
+    cd MyUnoApp.Skia.Gtk
+    dotnet run
+    ```
+Now let's try wasm
```
Comment (jeromelaban) [inline doc/articles/get-started-with-linux.md:108]:
```suggestion
Run the WebAssembly head with:
```

File: doc/articles/get-started-with-linux.md [hunk 4]
```diff
@@ -77,3 +77,46 @@ Now let's run the application:
...
+- Install the `dotnet new` templates:
+    ```bash
+    dotnet new -i Uno.ProjectTemplates.Dotnet::3.1-dev*
+    ```
+- Then create a new project using:
+    ```bash
+    dotnet new unoapp -o MyUnoApp
+    ```
+
+Now let's run the application:
```
Comment (jeromelaban) [inline doc/articles/get-started-with-linux.md:101]:
```suggestion
Run the GTK based application:
```

File: doc/articles/get-started-with-linux.md [hunk 5]
```diff
@@ -77,3 +77,46 @@ Now let's run the application:
     cd MyUnoApp.Skia.Gtk
     dotnet run
     ```
+    
+    
+    
+Using ArchLinux 5.8.14 or later / Manjaro:
+- update system and packages
```
Comment (jeromelaban) [inline doc/articles/get-started-with-linux.md:84]:
```suggestion
- Update system and packages
```

File: doc/articles/get-started-with-linux.md [hunk 6]
```diff
@@ -77,3 +77,46 @@ Now let's run the application:
...
+    ```bash
+    pacman -Syu
+    ```
+- install the necessary dependencies
```
Comment (jeromelaban) [inline doc/articles/get-started-with-linux.md:88]:
```suggestion
- Install the necessary dependencies
```

**Review by jeromelaban — COMMENTED** [review]
Thank you for your contribution!

Once the edits have been made, make sure to squash your commits, and use the conventional commits format. (the `docs:` prefix in your case).

**Comment by jeromelaban** [comment]
Thank for the updates, would you mind squashing your commits into a single one with the proper comment ? Thanks !

**Comment by Arwwarrr** [comment]
i dont know how

**Comment by jeromelaban** [comment]
You may want to take a look at this document: https://github.com/unoplatform/uno/blob/42a3dfc4ed7b6b44f89c95b38a8d5cf7abc78dd0/doc/articles/contributing/guidelines/pull-requests.md

**Pull request merged**